woocommerce/woocommerce-gateway-stripe
# Rename singleton to avoid 3rd party plugin conflict

# Changes proposed in this Pull Request:

Issue: https://github.com/woocommerce/woocommerce-gateway-stripe/issues/1375
Fixes #1375 

Description: Renames the singleton to avoid conflicts with the https://wordpress.org/plugins/woo-stripe-payment/ plugin

# Testing instructions

- Install and activate this branch
- Install and activate https://wordpress.org/plugins/woo-stripe-payment/ Edit: You must use a release < 3.2.8
- Ensure they are both able to be activated simultaneously without error

-------------------

(remove me)
